README.md

```diff
@@ -255,6 +255,46 @@ irods_http_api /path/to/config.json
 
 To stop the server, you can use **CTRL-C** or send **SIGINT** or **SIGTERM** to the process.
 
+### OpenID Connect
+~~To run the server with OpenID Connect, you must perform the following actions:~~
+
+Some Additional configuration is required to run the OpenID Connect portion of the HTTP API.
+Following are a few points of interest.
+
+#### OpenID Provider Requirements and HTTP API Configuration
+The OpenID Provider, at this moment, must support discovery via a well-known endpoint.
+This endpoint must specified in the `well_known_uri` OIDC configuration parameter.
+
+One should take care to ensure that `/.well-known/openid-configuration` is not included
+in the configuration parameter, as this is included automatically.
+
+One must also set the OIDC `config_host` and `port` parameter, so that the openid configuration
+may be gathered from the well-known endpoint on the HTTP API server's startup.
+The OpenID Provider must be running prior to starting the HTTP API server, otherwise, the HTTP API server
+will not be able to query the required information from the desired OpenID Provider.
+
+Additionally, the OIDC `redirect_uri` parameter must be set to the HTTP API's authentication endpoint.
+This is required, as the Authorization Code Grant needs to be redirected back to the HTTP API, to complete
+HTTP API token generation.
+
+~~At this point, if you were to start the HTTP API server, the server should be able to
+query the OpenID Provider for it's configuration.~~
+
+#### Add `irods_username` to the claims
+Currently, the server looks for an `irods_username` claim in the ID Token.
+This serves as the mapping mechanism for an OIDC User to an iRODS User.
+
+A user who authentices but does not have a `irods_username` mapped in their account
+will not have access to the API.
```

Comment:
please share the error that will be returned, to help the reader 'discover' that this is their problem.

Reply:
HTTP Response code is now in the docs!